SPEC.md

```diff
@@ -216,3 +229,6 @@ IPAM plugin examples:
 #### Notes
  - Routes are expected to be added with a 0 metric.
  - A default route may be specified via "0.0.0.0/0". Since another network might have already configured the default route, the CNI plugin should be prepared to skip over its default route definition.
+
+## Well-known Error Codes
+- `1` - Incompatabile CNI version
```

Comment:
are we going to define incompatibility? e.g. appc has this to say (not saying it's necessarily exactly what we want but to give you an idea):
```
The App Container specification aims to follow semantic versioning and retain forward and backwards compatibility within major versions. For example, if an implementation is compliant against version 1.0.1 of the spec, it is compatible with the complete 1.x series.

```